innersource-guidance-group.md

```diff
@@ -0,0 +1,43 @@
+## Title
+InnerSource Guidance Group
+
+## Patlet
+A company may require a mechanism to set broad governance and behavioral guidelines as part of the rollout and sustaining needs of an InnerSource initiative.
+
+## Problem
+Without a set of organizational standards Trusted Committers, and individual teams, are free to set their own standards, which is a step in the right direction.  However, team standards could be highly variant and further diverge over time as they evolve and adapt to new needs and forces.  This can lead to confusion and frustration as contributors submit code across team and functional boundaries. The business is interested establishing a set of agreed-upon standard operating procedures that guide teams while still providing sufficient autonomy to meet the parochial needs of individual teams.
+
+## Context
```

Comment:
In general suggest revising this section to think through what are the points of context that are essential for this pattern, specifically, as opposed to what is your own personal context, currently.